### PR TITLE
heartbeat added

### PR DESCRIPTION
**Pull request by rajanjoshigl**

### CODE REVIEW

File: lrlog/tests/lrlog_test.go [hunk 1]
```diff
@@ -0,0 +1,128 @@
+package tests
```
Comment (Himanshu8859) [inline lrlog/tests/lrlog_test.go]: file?

File: lrlog/lrlog.go [hunk 2]
```diff
@@ -0,0 +1,143 @@
+package lrlog
```
Comment (Himanshu8859) [inline lrlog/lrlog.go]: ??

**Review by Himanshu8859 — CHANGES_REQUESTED** [review]

File: lrlog/tests/lrlog_test.go (hunk 1 above)
Comment (rajanjoshigl) [reply lrlog/tests/lrlog_test.go]: This logging package is used by heartbeat package

File: lrlog/lrlog.go (hunk 2 above)
Comment (rajanjoshigl) [reply lrlog/lrlog.go]: This logging package is used by heartbeat package

**Pull request merged**